src/preferences/dialog/dlgprefinterfacedlg.ui

```diff
@@ -282,6 +282,19 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <spacer name="verticalSpacer">
+     <property name="orientation">
+      <enum>Qt::Vertical</enum>
+     </property>
+     <property name="sizeHint" stdset="0">
+      <size>
+       <width>20</width>
+       <height>40</height>
+      </size>
+     </property>
+    </spacer>
+   </item>
```

Comment:
This sets the min height to 40.

Let's do it like in DlgAutoDJ, where it can shrink to 0 but expands to push the stuff above up.
(it seems there are multiple ways to achive this)
```xml
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
    </spacer>
   </item>
```
Spacers are pretty incostinent in the pref dialogs, for example Normalization, Waveforms, Vinyl Control and Library also have this weird space at the bottom.